cyberark/secretless-broker
# WIP Adding search functionality

**What does this PR do?** 
Initial step for adding search functionality to the full site (docs, blog, pages) using Algolia.

**WIP**
- clear search field does not work

**Next steps**
- get OSS license for Algolia to remove bug on search bar
- search accessible from nav at all times, instead of navigating to new page

**How to test**
1. Click search icon in the top nav to navigate to /search and try to find some content
2. Confirm that your query is being highlighted in results
2. Confirm that appropriate results are appearing for your query
3. Confirm that links for results work
4. Check that clearing the search box clears the URL, searchbox, and results.

![screenshot_2018-10-15 search secretless](https://user-images.githubusercontent.com/123787/46954942-c1ff3a80-d05f-11e8-8a1d-dd947d14d8c9.png)
